### ISSUE TITLE
Example to create a timestamp (MinuteOfTheYear)

### ISSUE THREAD

**Issue by josephquzhou** [vlm/asn1c#269]
Is there an example code on how to create a timestamp (MinuteOfTheYear)? Thanks!

**Comment by DavidKelleySCSC** [comment]
For what industry or what ASN?   Do you mean this:
```

> MinuteOfTheYear ::= INTEGER (0..527040) 
>    -- the value 527040 shall be used for invalid

```
If so.... Starting with UTC time (not GPS time)... Multiply months by days in each, add current days in current month (recall a day rolls over at UTC midnight for DSRC) multiply by 86000 (secs in a day) add 1 more second **if** a leap year second was declared at the June-July epoch (one was not declared for this year, the announcement comes out in Jan) and if the needed time is past then. Divide by 60. You are done. You can skip the leap second on Dec 31st as it only effects the year for that final second.  If I work out some code I will post it on a MAP-SPAT form with to it a link here.  Regards, DCK Editor J2735

**Comment by josephquzhou** [comment]
Yes, it is MinuteOfTheYear in J2735. I thought there might be an existing function to use. I will appreciate it if you can post some code. Thanks!
